manifests/config.pp

```diff
@@ -90,6 +90,7 @@
     globus_connect_config { 'MyProxy/ServerBehindNAT': value => $globus::myproxy_server_behind_nat }
     globus_connect_config { 'MyProxy/CADirectory': value => $globus::myproxy_ca_directory }
     globus_connect_config { 'MyProxy/ConfigFile': value => $globus::myproxy_config_file }
+    globus_connect_config { 'MyProxy/CaSubjectDN': value => $globus::myproxy_ca_subject_dn }
```

Comment:
Would need to wrap this in if statement so only manage this resource if `$globus::myproxy_ca_subject_dn` is not `undef`

Reply:
Good point, I should have tested more thoroughly!